docs/src/pages/components/heading/index.page.mdx

```diff
@@ -1,7 +1,6 @@
 ---
 title: Heading
-description: This is a separate primitive from Text for semantic purposes. The Heading primitive maps to an <h*> tag, and Text maps to a <p> tag.
-themeSource: packages/ui/src/theme/tokens/components/heading.ts
```

Comment:
For my knowledge, what is `themeSource` and why is it removed here?

Reply:
It should be removed, thanks for the catch. It's used to render a button that links to the heading design tokens

Reply:
fixed

Reply:
😉  You said it should be removed at first

Reply:
I catch things without knowing what they are ;)